src/modules/Crowbar.rb

```diff
@@ -118,6 +93,13 @@ def json2hash(file_name)
       ret
     end
 
+    # read given yaml file and return the content as a map
+    def yaml2hash(file_name)
+      ret = YAML.load(File.read(file_name))
```

Comment:
Is it an issue to have `nil`? When I was testing, it didn't seem to be a problem, but I might have missed something.